modules/mod_ginger_edit/templates/_action_ginger_dialog_new_rsc_tab.tpl

```diff
@@ -1,5 +1,5 @@
 {% with redirect|default:(dispatch=="ginger_edit")|if:[]:`ginger_edit` as redirect %}
-{% with callback|default:(dispatch=="ginger_edit")|if:"zAdminConnectDone":"" as callback %}
+{% with callback|default:"" as callback %}
```

Comment:
Fred+1.

Before it was defaulted to "zAdminConnectDone" if it was from a "ginger_edit" page. Otherwise it was "". Since there is a live reload within ginger_edit there is no need for "zAdminConnectDone" to be executed here, so default "". Custom callbacks should keep working, and ginger_edit is changed. Only if sites still use the old version it could be problem?